tools/SynchronizeData.py

```diff
@@ -610,10 +644,14 @@ def save(self, header, data, output_file=None):
 
 if __name__ == "__main__":
```

Comment:
I would not include the changes made here if they contain files path on a specific computer